interface/patient_file/encounter/diagnosis.php

```diff
@@ -1,312 +0,0 @@
-<?php
```

Comment:
He has a return with that file name. I am not sure what he is doing, but it is not pointing to the interface/patient_file/encounter/ directory. 

Tell me do y'all want me to put it back?  @tmccormi @aethelwulffe